tests/app-interface/data/dependencies/vault/permissions/oidc/master/app-sre-vault-admin.yml

```diff
@@ -4,7 +4,7 @@ $schema: /access/oidc-permission-1.yml
 labels:
   service: vault.local
 
-name: app-sre-vault-admin
+name: app-sre-vault-admin-primary
```

Comment:
Change is fine, but why?

Reply:
When creating a new `data.json` file, it was failing due to duplicate names.  This change made it able to generate without failing.